src/main/java/com/actiontech/dble/util/ExecutorUtil.java

```diff
@@ -66,7 +66,7 @@ private static NameableExecutor createCached(String name, int size, boolean isDa
      */
     public static NameableExecutor createCached(String name, int size, int maxSize, ThreadChecker checker) {
```

Comment:
this api is only made for timeExecuter. 
should use a special function name to distinguish.

Reply:
get